commit 4adeb7541a2c5aaf581478251c450e1184addfa0
Author: Alejandro Alvarez Ayllon <a.alvarezayllon@gmail.com>
Date:   Mon Dec 20 10:02:35 2021 +0100

    failUnlessRaises were deprecated in Python 3.1
    
    and they will be removed in Python 3.11

diff --git a/cmake/scripts/EnvConfig/TestEnvOps.py b/cmake/scripts/EnvConfig/TestEnvOps.py
index 0d2f3d0b..02a44da8 100644
--- a/cmake/scripts/EnvConfig/TestEnvOps.py
+++ b/cmake/scripts/EnvConfig/TestEnvOps.py
@@ -239,35 +239,35 @@ class Test(unittest.TestCase):
         self.assertTrue(isinstance(control.var('loc2'), Variable.Scalar))
 
         control.declare('MY_PATH', 'list', False)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'MY_PATH', 'list', True)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'MY_PATH', 'scalar', True)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'MY_PATH', 'scalar', True)
 
         control.declare('loc', 'list', True)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'loc', 'list', False)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'loc', 'scalar', True)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'loc', 'scalar', True)
 
         control.declare('myVar2', 'scalar', False)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'myVar2', 'list', False)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'myVar2', 'list', True)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'myVar2', 'scalar', True)
 
         control.declare('loc2', 'scalar', True)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'loc2', 'list', False)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'loc2', 'list', True)
-        self.failUnlessRaises(
+        self.assertRaises(
             Variable.EnvError, control.declare, 'loc2', 'scalar', False)
 
     def testDelete(self):
